src/binder/expression/mod.rs

```diff
@@ -133,6 +133,26 @@ impl BoundExpr {
         visitor.visit_expr(self);
         visitor.0
     }
+
+    pub fn format_name(&self, child_schema: &Vec<ColumnDesc>) -> String {
+        println!("{:?}", self);
```

Comment:
Please don't print extra things to the screen. Also, would you please add unit tests for this function?